src/status_im2/contexts/chat/messages/content/pin/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns status-im2.contexts.chat.messages.content.pin.view
+  (:require [react-native.core :as rn]
+            [status-im2.contexts.chat.messages.content.pin.style :as style]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [utils.re-frame :as rf]
+            [i18n.i18n :as i18n]
+
+            ;;TODO (flexsurfer) refactor and move
```

Comment:
same for TODO's here. let's create issues so other developers are enabled to work on these.